### ISSUE TITLE
Matching query using WithArgs behavior.

### ISSUE THREAD

**Issue by eklmv** [pashagolub/pgxmock#114]
If query in tested code got some arguments, but in test we do not use `.WithArgs`, or even providing no args with `.WithArgs()` it still matches query. Is it correct behavior? Or it should rise arguments do not match error, that shows that no arguments was expected but actually received some? 
Example test passing:
```
func Test_QueryMatchingMissingWithArgs(t *testing.T) {
	mock, err := pgxmock.NewConn()
	if err != nil {
		t.Errorf("an error '%s' was not expected when opening a stub database connection", err)
	}

	mock.ExpectQuery("SELECT id FROM").WithArgs().WillReturnRows(pgxmock.NewRows([]string{"id"}).AddRow(1))

	var id int
	row := mock.QueryRow(context.Background(), "SELECT id FROM users WHERE name=$1", "john")
	err = row.Scan(&id)
	if err != nil {
		t.Errorf("error '%s' was not expected", err)
	}

	if err := mock.ExpectationsWereMet(); err != nil {
		t.Errorf("there were unfulfilled expectations: %s", err)
	}
}
```

**Comment by pashagolub** [comment]
Is this fixed by #115?

**Comment by eklmv** [comment]
Yes, thought github will catch it from merge title.
